.github/workflows/notify_repositories.yaml

```diff
@@ -0,0 +1,39 @@
+name: Notify repositories of changes in definitions
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "definitions/**"
+
+jobs:
+  notify-repositories:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        repository: ${{ fromJson(vars.NOTIFY_REPOS) }}
+
+    steps:
+      - name: Send repository_dispatch to ${{ matrix.repository }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+        run: |
+          TAG="${{ github.event.release.tag_name }}"
+          REPO="${{ matrix.repository }}"
+          echo "Notifying repository: $REPO"
```

Comment:
Remove the `curl --version` line as it adds unnecessary noise to the logs without providing value.